### PR TITLE
#438 Frontend type

### PR DESCRIPTION
**Pull request by jojopirker**
Added FrontendType to database and class, also changed the readme a bit to have everything at one place

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -20,3 +33,7 @@ REDIS_PORT=6379
 Have a look into the main `README.md` file for more information on how to set up
 the backend for development. Use the scripts within the
 scripts/backend-development folder to run the BE API locally.
+
+## Alembic
+
+To the database run `alembic revision -m "..."` ("..." is what you did) in the `/backend` directory. Then edit the newly created file. See [here](https://alembic.sqlalchemy.org/en/latest/tutorial.html) for more information.
```
Comment (andreaskoepf) [inline backend/README.md:39]: This should be `alembic revision --autogenerate -m "..."` ... and I think at the beginning of the sentence it should be "To create an Alembic database migration script after sql-models were modified run `alembic revision --autogenerate -m "..."`"

**Comment by andreaskoepf** [comment]
Regarding `pre-commit` please see [CONTRIBUTING.md#pre-commit](https://github.com/LAION-AI/Open-Assistant/blob/9881b9d8cbaf3a03df4b8e2c9eb55eab08b15dac/CONTRIBUTING.md#pre-commit)

**Comment by jojopirker** [comment]
I always forget pre-commit  🤦

**Comment by andreaskoepf** [comment]
> I always forget pre-commit facepalm

Thanks for the fixes. Consider running `pre-commit install` in the OA dir to install the hook - then you can't forget it anymore. ;-)

**Review by andreaskoepf — APPROVED** [review]
Looks good, thanks!

**Pull request merged**